styles.css

```diff
@@ -654,6 +654,154 @@ li {
   border-radius: 50%;
 }
 
+.contact-container {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+.contact-wrapper {
+  width: 327px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  height: 870px;
+  left: -1px;
+  top: 5887px;
+  border-radius: 0;
+  padding: 120px 24px 120px 24px;
+}
+
+.contact-header {
+  height: 262px;
+  width: 327px;
+  left: 24px;
+  top: 120px;
+  border-radius: 0;
+  font-family: 'Inter', sans-serif;
+  font-size: 2rem;
+  line-height: 44px;
+  font-weight: 700;
+  text-align: center;
+  color: #172b4d;
+  margin-bottom: 70px;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  gap: 14px;
+  width: 327px;
+  left: 24px;
+  top: 440px;
+  border-radius: 0;
+}
+
+.contact-wrapper input::placeholder {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #979493;
+}
+
+.contact-wrapper form input {
+  border: none;
+  border-bottom: 1px solid #dbd8d7;
+  padding: 14px 17px;
+  margin-top: 1.2rem;
+  width: 327px;
+}
+
+.contact-fields textarea {
+  margin: 1.2rem 0;
+}
+
+.contact-wrapper form textarea {
+  background: #fbf8f7;
+  border: none;
+  padding: 10px 10px;
+  height: 114px;
+  width: 327px;
+}
+
+textarea::placeholder {
+  font-family: 'Inter', sans-serif;
+  font-style: italic;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #3c3a39;
+}
+
+.btn-contact {
+  width: 131px;
+  height: 48px;
+  align-self: center;
+  font-family: "Inter", sans-serif;
+  border: 1px solid transparent;
+  line-height: 24px;
+  font-size: 17px;
+  padding: 12px;
+  font-weight: 700;
+  background-color: #f55800;
+  color: #dbd8d7;
+  margin-left: 100px;
+  margin-top: 0;
+}
+
+.footer {
+  gap: 14px;
+  display: flex;
+  width: 200px;
+  height: 72px;
+  align-items: center;
+  margin-left: 88px;
+  margin-right: 87px;
+  margin-bottom: 40px;
+  flex-direction: column;
+  justify-content: inline-block;
+}
+
+.hori-div {
+  background-color: gray;
+  width: 375px;
+  margin-top: -10px;
+  align-items: center;
+}
```

Comment:
- [ ] Kindly make sure to stay consistent with the given Figma design when choosing colors as I noticed the background color of the horizontal line in your design should be `#DFE1E6` and not gray so make sure to fix this.
- Your design:
  ![image](https://github.com/Zuwairanajma/Portfolio-Projects/assets/104968236/8211ca1d-d4a1-4ae1-8d23-3d9d30702bd8)
- Figma design:
  ![image](https://github.com/Zuwairanajma/Portfolio-Projects/assets/104968236/a93dbf61-147d-4577-a88c-7f4ac748e2e0)